periphery/src/BorrowManager.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.15;
+
+import {ERC20, SafeTransferLib} from "solmate/utils/SafeTransferLib.sol";
+import "v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+import {Uniswap} from "aloe-ii-core/libraries/Uniswap.sol";
+import {Factory} from "aloe-ii-core/Factory.sol";
+import {IManager, Borrower} from "aloe-ii-core/Borrower.sol";
+import {Lender} from "aloe-ii-core/Lender.sol";
+
+contract BorrowManager is IManager {
+    using SafeTransferLib for ERC20;
+
+    Factory public immutable FACTORY;
+
+    constructor(Factory _factory) {
+        FACTORY = _factory;
+    }
+
+    /* solhint-disable code-complexity */
```

Comment:
Can you make this disable more specific, and re-enable the code-complexity check after the problematic section of code?

Reply:
Will do, just a heads up, I used the disable from FrontendManager